app/tests/jest/script/wrapper.js

```diff
@@ -1,12 +1,15 @@
 import _ from 'lodash'
+import Vuex from 'vuex'
 import VueMaterial from 'vue-material'
 import store from '@/store'
```

Comment:
We also already import store here, is this redundant?

Reply:
Yes it is, I added it during testing tweaks for the wrapper and forgot to remove